spec/models/workshop_spec.rb

```diff
@@ -16,4 +16,16 @@
     end
   end
 
+  it "is valid with a title" do
+    workshop = Workshop.new(title:"Workshop Title")
+    workshop.valid?
+    expect(workshop.errors).to_not have_key(:title)
+  end
+
+  it "is valid with a description" do
+    workshop = Workshop.new(description:"A short description")
+    workshop.valid?
+    expect(workshop.errors).to_not have_key(:description)
```

Comment:
since you want the whole thing to be valid, not only the attribute `:description`, I'd like to see something like this: `expect(workshop.valid?).to eq(true)`